src/net.cpp

```diff
@@ -580,6 +580,26 @@ bool CNode::IsBlockRelayOnly() const {
     return (ignores_incoming_txs && !HasPermission(PF_RELAY)) || !IsAddrRelayPeer();
 }
 
+std::string CNode::ConnectionTypeAsString() const
```

Comment:
for #19725 please create a new file `doc/release-notes-19725.md`
 and put missing changes there.

In this case backport 19725 is not partial anymore, can remove this mark, is it?